compass/landice/mesh.py

```diff
@@ -60,3 +61,44 @@ def gridded_flood_fill(field):
         lastSearchList = newSearchList
 
     return floodMask
+
+
+def set_geom_points_and_edges(xmin, xmax, ymin, ymax):
```

Comment:
I propose to rename this to `set_rectangular_geom_points_and_edges` or similar.  I suspect defining the region to mesh by a rectangle is what we'll always do for convenience, but in principle we could create any custom polygon shape to mesh inside of.  When we get to ultra high res or possibly if we try to mesh to exact boundaries, we may introduce other, similar functions.

Reply:
Renamed in [68a8205](https://github.com/MPAS-Dev/compass/pull/319/commits/68a82051daed908e1ebfbb3051b4a862c27aaa59)